workflow/scripts/sequence_logos.py

```diff
@@ -103,7 +103,7 @@
     )
     logo = logomaker.Logo(
         prob_mat,
-        ## fade_probabilities=True, ## will fade the smaller probabilities
+        fade_probabilities=True, ## will fade the smaller probabilities
```

Comment:
What does it mean?
Does this mean that we will only see the single highest letter? That might **not** be desireable: many PWMs are not polarised and it could happend that there are 2 major letters vs 2 not-noticeable letters (at a given position). But still - we would like to see what are the two which domnate the rest.

Reply:
No, even if one is of lesser probability, it will still be visible. For example, in the attached image the probability of C is 20%.
![motif_HNRNPF_912](https://user-images.githubusercontent.com/51164789/89658525-49c8cf80-d8ec-11ea-8b0d-53055a86b44e.png)

Reply:
Ah, OK, so I get it that the *fading* is related to transparency?